### DISCUSSION TITLE
Is it possible to retrieve Seller Profile data from Amazo SP API?

### DISCUSSION THREAD

**Discussion by wolendranh** [saleweaver/python-amazon-sp-api#92] [Q&A]
Hi! 
I went through multiple docs and not sure if it is possible to retrieve Seller Profile data from Amazo SP API. We receive Partner_ID after Seller authorise application. Can we get Seller profile data somehow?

**Comment by yberezkin** [comment]
What kind of Data do you looking for? What Seller Profile Data should contain?

**Reply by wolendranh** [reply]
For example data provided by `https://api.amazon.com/user/profile`. But when I make request with token that I get for seller i receive 401 error, `b'{"error_description":"The access token provided does not have required scope","error":"insufficient_scope"}'`

**Reply by wolendranh** [reply]
@yberezkin Do you have some suggestion how I can proceed?

**Reply by yberezkin** [reply]
As far as I know `https://api.amazon.com/user/profile` Is not part of SellerPartner API.

What data do you want to retrieve specially about Seller?

**Reply by wolendranh** [reply]
Seller profile. Email etc. Currently except `Partner_ID` there is nothing I can get. Is it possible to get something more except `Partner_ID `?

**Comment by saleweaver** [comment]
No, there’s no way to get this data from SP-API sources.


Am 11. März 2021, 12:26 +0100 schrieb Yura Hulpa ***@***.***>:


Seller profile. Email etc. Currently except Partner_ID there is nothing I can get. Is it possible to get something more except Partner_ID ?
